.github/workflows/release.yaml

```diff
@@ -0,0 +1,41 @@
+name: Create Release
+
+on:
+  push:
+    tags:
+      - '*.*.*' # Trigger on any tag that starts with 'v'
```

Comment:
might want to modify the coment here, because will trigger a release w/ a tag that follow the folllowing formate *.*.* right?